.pre-commit-config.yaml

```diff
@@ -112,8 +112,15 @@ repos:
               README.md
           )$
 
+  - repo: https://github.com/pre-commit/mirrors-prettier
+    rev: v3.0.3
+    hooks:
+      - id: prettier
+        # https://prettier.io/docs/en/options.html#print-width
+        args: ["--print-width=120"]
```

Comment:
Should we use https://prettier.io/docs/en/cli#--ignore-path instead of adding a new file to the root of the project?

Reply:
Good point, I was using extra file as we did in template (it was needed there to share ignore between precommit amd stands JS build tools)

Will update it...

Reply:
well trying it with `--ignore-path="**/*.md"` and seems not working at all

Reply:
It takes a path to an ignore file, not a pattern :)